spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -83,14 +86,17 @@ trait CommonJobExecution {
     dao.authenticate()
     val dataset = dao.getDataset(cmd.datasetName, cmd.datasetVersion)
     val reportVersion = getReportVersion(cmd, dataset)
-    val pathCfg = getPathCfg(cmd, dataset, reportVersion)
+    val pathCfg = getPathConfig(cmd, dataset, reportVersion)
+
+    val (inputPath, outputPath) = getInputOutputPaths(cmd, fsUtils, pathCfg)
+
+    log.info(s"input path: $inputPath")
+    log.info(s"output path: $outputPath")
 
-    log.info(s"input path: ${pathCfg.inputPath}")
-    log.info(s"output path: ${pathCfg.outputPath}")
     // die if the output path exists
-    validateForExistingOutputPath(fsUtils, pathCfg)
+    validateForExistingOutputPath(fsUtils, outputPath)
```

Comment:
Would make this  abstract here, and implemented in successors. (Combined job could call both ancestor's version)

Reply:
Good point